File: atcoder/abc192/abc192_b.py
inp = input()

inp = list(inp)
tam = len(inp)


cumple = True


for i in range(0,tam,2):


    
    islow2 = inp[i].islower()
    
    if islow2 != True:
        cumple = False
        break
    
    
if cumple == True:
    
    for j in range(1,tam,2):
        
        isUpp = inp[j].isupper()
        
        if isUpp != True:
            cumple = False
            break
    
if cumple == True:
    print("Yes")
else:
    print("No")

File: atcoder/abc192/abc192_b.cpp
#include <iostream>
//data structures
#include <array>
#include <map>
#include <vector>
#include <string>
#include <algorithm>
#include <unordered_map>
#include <unordered_set>
#include <bitset>
#include <queue>
#include <typeinfo>
//other
#include <bits/stdc++.h>

using namespace std;

void printArr(int a[], int sz) {
    for (int i=0; i<sz; i++)
        cout << a[i] << " ";
    cout << "\n";
}

int main() {
    string st;
    cin >> st;
    bool hard = true;
    for (int i=0; i<st.size() && hard; i++) {
        char c = st[i];
        if (i%2 == 0) { // Odd
            if (isupper(c))
                hard = false;
        } else { // Oddn't
            if (islower(c))
                hard = false;
        }
    }
    string out = hard ? "Yes" : "No";
    cout << out << "\n";
    return 0;
}